derive/src/pymodule.rs

```diff
@@ -399,28 +388,46 @@ pub fn impl_pymodule(attr: AttributeArgs, item: Item) -> Result<TokenStream2, Di
 
     let items = content
         .iter_mut()
-        .filter_map(|item| match item {
-            Item::Fn(syn::ItemFn { attrs, sig, .. }) => Some(ItemIdent {
-                typ: ItemType::Fn,
-                attrs,
-                ident: &sig.ident,
-            }),
-            Item::Struct(syn::ItemStruct { attrs, ident, .. }) => Some(ItemIdent {
-                typ: ItemType::Struct,
-                attrs,
-                ident,
-            }),
-            Item::Enum(syn::ItemEnum { attrs, ident, .. }) => Some(ItemIdent {
-                typ: ItemType::Enum,
-                attrs,
-                ident,
-            }),
-            Item::Const(syn::ItemConst { attrs, ident, .. }) => Some(ItemIdent {
-                typ: ItemType::Const,
-                attrs,
-                ident,
-            }),
-            _ => None,
+        .filter_map(|item| {
+            let item = match item {
+                Item::Fn(syn::ItemFn { attrs, sig, .. }) => ItemIdent {
+                    typ: ItemType::Fn,
+                    attrs,
+                    ident: &sig.ident,
+                },
+                Item::Struct(syn::ItemStruct { attrs, ident, .. }) => ItemIdent {
+                    typ: ItemType::Struct,
+                    attrs,
+                    ident,
+                },
+                Item::Enum(syn::ItemEnum { attrs, ident, .. }) => ItemIdent {
+                    typ: ItemType::Enum,
+                    attrs,
+                    ident,
+                },
+                Item::Const(syn::ItemConst { attrs, ident, .. }) => ItemIdent {
+                    typ: ItemType::Const,
+                    attrs,
+                    ident,
+                },
+                Item::Use(syn::ItemUse { attrs, tree, .. }) => {
+                    let ident = match tree {
+                        UseTree::Path(path) => match &*path.tree {
+                            UseTree::Name(name) => &name.ident,
+                            UseTree::Rename(rename) => &rename.rename,
+                            _ => return None,
```

Comment:
It'd probably be nice to support multiple imports `use foo::{BAR, BAZ, QUX};`, though that would probably require a bit of restructuring other code to do so

Reply:
I am thinking of overhaul of derive flow 😞